components/providers/confirm-provider.tsx

```
@@ -0,0 +1,134 @@
+									<h3 className="text-lg font-semibold text-gray-900 dark:text-gray-100 mb-2">
+										{options.title}
+									</h3>
+								)}
+								<p className="text-sm text-gray-600 dark:text-gray-400">
+									{options.message}
+								</p>
+							</div>
+							<button
+								onClick={handleCancel}
+								className="flex-shrink-0 text-gray-400 hover:text-gray-600 dark:hover:text-gray-200 transition-colors"
+							>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)